Makefile

```diff
@@ -8,23 +8,27 @@ WEAVE_VERSION=git-$(shell git rev-parse --short=12 HEAD)
 WEAVER_EXE=weaver/weaver
 WEAVEDNS_EXE=weavedns/weavedns
 WEAVETOOLS_EXES=tools/bin
+WEAVERENDEZ_EXE=cmd/rendezvous/weaverendezvous
 WEAVER_IMAGE=zettio/weave
 WEAVEDNS_IMAGE=zettio/weavedns
 WEAVETOOLS_IMAGE=zettio/weavetools
+WEAVERENDEZ_IMAGE=zettio/weaverendezvous
 WEAVER_EXPORT=/var/tmp/weave.tar
 WEAVEDNS_EXPORT=/var/tmp/weavedns.tar
 WEAVETOOLS_EXPORT=/var/tmp/weavetools.tar
+WEAVERENDEZ_EXPORT=/var/tmp/weaverendezvous.tar
 
-all: $(WEAVER_EXPORT) $(WEAVEDNS_EXPORT) $(WEAVETOOLS_EXPORT)
+all: $(WEAVER_EXPORT) $(WEAVEDNS_EXPORT) $(WEAVETOOLS_EXPORT) $(WEAVERENDEZ_EXPORT)
 
 update:
 	go get -u -f -v -tags -netgo ./$(dir $(WEAVER_EXE)) ./$(dir $(WEAVEDNS_EXE))
 
-$(WEAVER_EXE) $(WEAVEDNS_EXE):
+$(WEAVER_EXE) $(WEAVEDNS_EXE) $(WEAVERENDEZ_EXE):
 	go get -tags netgo ./$(@D)
 	go build -ldflags "-extldflags \"-static\" -X main.version $(WEAVE_VERSION)" -tags netgo -o $@ ./$(shell dirname $@)
-	@strings $@ | grep cgo_stub\\\.go >/dev/null || { \
+	@ldd $@ 2>/dev/null | grep "not a dynamic executable" >/dev/null || { \
```

Comment:
This change was made because the previous condition didn't seem right either: even after recompiling with the "netgo" flag set, the `weaverendezvous` binary had a `"cgo_stub"` string. I think there are no dependencies that link with cgo generated libraries either... So any suggestion?